volatility3/framework/plugins/linux/modxview.py

```diff
@@ -89,43 +94,50 @@ def run_modules_scanners(
         )
 
     def _generator(self):
-        kernel_name = self.config["kernel"]
+        kernel = self.context.modules[self.config["kernel"]]
 
-        kernel = self.context.modules[kernel_name]
+        wanted_gatherers = [
+            linux_utilities_modules.ModuleGathererLsmod,
```

Comment:
So... I can't tell if these should be versioned individually (since they *should* be part of the main list, but...)

Reply:
That is a very good catch and would set a very bad precedent if we didn't do this. I will update that shortly.